.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,114 +1,133 @@
+        - Installer
+        - Keyboard Manager
+        - Mouse Utilities
+        - Mouse Without Borders
+        - Paste as Plain Text
+        - Peek
+        - PowerRename
+        - PowerToys Run
+        - Quick Accent
+        - Registry Preview
+        - Screen ruler
+        - Settings
+        - Shortcut Guide
+        - System tray interaction
+        - TextExtractor
+        - Video Conference Mute
+        - Welcome / PowerToys Tour window
+        - General / Other
 
-- type: dropdown
-  attributes:
-    label: Running as admin
-    description: Are you running PowerToys as Admin?
-    options:
-      - "Yes"
-      - "No"
+  - type: dropdown
+    attributes:
+      label: Running as Administrator?
+      options:
+        - "Yes"
+        - "No"
 
-- type: dropdown
-  attributes:
-    label: Area(s) with issue?
-    description: What things had an issue? Check all that apply.
-    multiple: true
-    options:
-      - General
-      - Always on Top
-      - Awake
-      - ColorPicker
-      - Command not found
-      - Crop and Lock
-      - Environment Variables
-      - FancyZones
-      - FancyZones Editor 
-      - File Locksmith 
-      - "File Explorer: Preview Pane"
-      - "File Explorer: Thumbnail preview"
-      - Hosts File Editor
-      - Image Resizer
-      - Installer
-      - Keyboard Manager
-      - Mouse Utilities
-      - Mouse Without Borders
-      - Paste as Plain Text
-      - Peek
-      - PowerRename
-      - PowerToys Run
-      - Quick Accent
-      - Registry Preview
-      - Screen ruler
-      - Settings
-      - Shortcut Guide
-      - System tray interaction
-      - TextExtractor
-      - Video Conference Mute
-      - Welcome / PowerToys Tour window
-  validations:
-    required: true
+  - type: textarea
+    id: reproduce
+    validations:
+      required: true
+    attributes:
+      label: Steps to reproduce
+      description: Having detailed steps helps us to reproduce the bug.
 
-- type: textarea
-  attributes:
-    label: Steps to reproduce
-    description: We highly suggest including screenshots and a bug report log (System tray > Report bug).
-    placeholder: Having detailed steps helps us reproduce the bug. 
-  validations:
-    required: true
+  - type: textarea
+    id: behavior
+    validations:
+      required: true
+    attributes:
+      label: ❌ Actual behavior
+      description: Describe what happened, as detailed as possible. Screenshots/GIFs can be very helpful. Include the Bug Report here.
 
-- type: textarea
-  attributes:
-    label: ✔️ Expected Behavior
-    placeholder: What were you expecting?
-  validations:
-    required: false
+  - type: checkboxes
+    attributes:
+      label: Include the Bug Report
+      description: "Right-click on our system tray icon and select **Bug Report**. Then upload the zip file from your desktop: Paste, drop or click to add files."
+      options:
+        - label: I have attached the Bug Report file
+          required: true
```

Comment:
I think the previous text was sufficient. There are a lot of scenarios where including a bug report is not an option (installer issues for example).

Reply:
I suppose I can make required: false.